package.json

```diff
@@ -119,9 +120,11 @@
     "lodash": "4.17.11",
     "nodelist-foreach-polyfill": "^1.2.0",
     "prop-types": "^15.6.0",
+    "react-async-script": "^1.0.1",
```

Comment:
Why was this added? We don't seem to use it anywhere. I know it was moved from `devDependencies`, but there is no reference to this package in the whole project.

Reply:
It's also peer dependency

Reply:
I think it's recaptcha package